omnibus/files/private-chef-ctl-commands/secrets.rb

```diff
@@ -42,16 +51,15 @@ def confirm_continue!(message)
   end
 end
 
-def set_secret(group, key, secret)
+def set(group, key, secret)
```

Comment:
The only thing to keep in mind is that these all get `eval`d into the ChefServerCtl class - so `set` will become a member function of the class.  

If there's any chance of another command (there are none) or existing function of the same name (I'm less sure about this without looking closer), odd things will happen without clear indication of what went wrong. 

This one is generic enough that we're probably better off keeping it as `set_secret`, even if only to prevent accidental future collisions.